### DISCUSSION TITLE
Installable Web Application(PWA)

### DISCUSSION THREAD

**Discussion by marvinhosea** [dailydotdev/daily#231] [Feature Requests]
I think installable web application will allow users to install the web application as app which will give them the convenience of using the web app as a normal app and also receive notifications

**Comment by esau-morais** [comment]
> I think installable web application will allow users to install the web application as app which will give them the convenience of using the web app as a normal app and also receive notifications

Daily.dev already created a Progressive Web App called Daily.go. But how you described, using Web Push Notifications, can be a good feature. If you are updated, there's a upcoming version of Daily.dev (web, current beta) that can replace the PWA mentioned (Daily.go). Now, let's see with the maintainers say about it :)

**Comment by idoshamun** [comment]
Hey @marvinhosea,

Thanks for dropping here your idea :)
As @esau-morais said, a PWA is definitely on our roadmap, and we expect to launch it at the beginning of 2021.
Hopefully, we will slowly launch new features along the way so you can enjoy them as early as possible.

**Comment by nimrodkra** [comment]
@marvinhosea  quick update on this thread: we released a beta version of daily.dev web app. It means daily.dev is now available as a PWA for mobile as well ✨ Feel free to try it here: https://app.daily.dev

Let us know what you think :)
